tests/test_merging.py

```diff
@@ -40,3 +43,18 @@ def test_merge_categories():
             "biolink:DiseaseOrPhenotypicFeature",
         ],
     )
+
+
+def test_merge_qedges():
+    """Test merge_qedges"""
+    with open("tests/test_jsons/test_merge_qedges1.json", "r") as file:
+        qedges1 = json.load(file)
+    with open("tests/test_jsons/test_merge_qedges2.json", "r") as file:
+        qedges2 = json.load(file)
+
+    merged_qedges = merge_qedges(qedges1, qedges2)
+
+    with open("tests/test_jsons/test_merge_qedges_success.json", "r") as file:
+        correct_merged_qedges = json.load(file)
+
+    assert merged_qedges == correct_merged_qedges
```

Comment:
Can we add a test that makes sure we hit the conflicting-value error?

Reply:
Yes, I can do that